apps/03TicTacToe.js

```diff
@@ -22,23 +22,36 @@ function printBoard() {
 }
 
 function horizontalWin() {
-    // Your code here
+  return ((board[0][0]===playerTurn && board[0][1]===playerTurn && board[0][2]===playerTurn)||
+  (board[1][0]===playerTurn && board[1][1]===playerTurn && board[1][2]===playerTurn)||
+  (board[2][0]===playerTurn && board[2][1]===playerTurn && board[2][2]===playerTurn));
 }
 
 function verticalWin() {
-    // Your code here
+  return ((board[0][0]===playerTurn && board[1][0]===playerTurn && board[2][0]===playerTurn)||
+  (board[0][1]===playerTurn && board[1][1]===playerTurn && board[2][1]===playerTurn)||
+  (board[2][0]===playerTurn && board[2][1]===playerTurn && board[2][2]===playerTurn));
 }
 
-function diagonalWin() {
-    // Your code here
+function diagonalWin () {
+  return ((board[0][0]===playerTurn && board[1][1]===playerTurn && board[2][2]===playerTurn)||
+  (board[0][2]===playerTurn && board[1][1]===playerTurn && board[2][0]===playerTurn));
 }
```

Comment:
No **if** statement needed?! Whoa! Nice job collapsing all your logic in one return statement!

![wizard](https://cdn.meme.am/instances/64335184.jpg)